packages/bootleg-tokens/contracts/ERC721/ERC721.sol

```diff
@@ -0,0 +1,285 @@
+pragma solidity ^0.5.0;
+
+import "./IERC721.sol";
+import "./IERC721Receiver.sol";
```

Comment:
It's curious that you needed this one. When I made these changes locally I didn't. 

What contract needed it?

Reply:
I guess I was on autopilot. Not sure why.